modules/standard/Debugger.chpl

```diff
@@ -20,20 +20,49 @@
 
 /* Provides a collection of useful debugging utilities.
 
-The Debugger module currently only provides a single function,
-:proc:`breakpoint`. This function can be used to set a breakpoint in the code
-that will be hit when the program is run under a debugger.
+The Debugger module currently provides two functions: :proc:`breakpoint` and
+:proc:`debugTrap`. Both functions can be used to set breakpoints in your code
+when using a debugger. Users should prefer using the :proc:`breakpoint`
+function, as it is more portable and easier to use.
+
+The :proc:`debugTrap` function is a lower-level function that raises a debug
+trap exception. This function is useful for testing purposes, but can crash the
+program if used and no debugger is attached.
 */
 @unstable(category="experimental", reason="The Debugger module is unstable due to its experimental behavior")
 module Debugger {
   /*
-    Sets a breakpoint at this point in execution if compiled with `-g`.
+    Sets a breakpoint at this point in execution.
 
-    .. warning::
-       If code uses :proc:`breakpoint` and is compiled with `-g`,
-       the program will not be runnable outside of a debugger
+    This only works if a breakpoint is set on the 'debuggerBreakHere' function
+    with ``b debuggerBreakHere``. If using ``--gdb`` or ``--lldb``, this
+    is done automatically.
   */
   inline proc breakpoint {
-    __primitive("breakpoint");
+    extern proc debuggerBreakHere();
+    debuggerBreakHere();
+  }
+
+  // TODO: This should be a private config var (to force qualified access)
+  // but that suppresses the docs for it.
```

Comment:
Is this comment out of date? This config has ``private``

Reply:
No. It should be `private`, the todo is about exposing the documentation for this config (because `private` suppresses documentation)